middleware/isUnderMaintenance.js

```diff
@@ -0,0 +1,21 @@
+const Organization = require("../models/organization");
+const createError = require("http-errors");
+
+module.exports = async (req, res, next) => {
+  const organization = await Organization.findOne({}).lean();
+  if (organization && organization.isUnderMaintenance) {
+    if (req.headers["access-level"] == -1 || (req.currentUser && req.currentUser.isAdmin)) {
```

Comment:
https://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions

Reply:
Since it is no longer recommended or is a necessity to prefix custom headers with X-; this name will be good to go